main.go

```diff
@@ -211,7 +211,12 @@ func main() {
 		)
 
 		r.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-			http.Redirect(w, r, "/api/metrics/v1/graph", http.StatusMovedPermanently)
+			r.URL.Path = "/api/metrics/v1/graph"
+			http.Redirect(w, r, r.URL.String(), http.StatusMovedPermanently)
+		})
+		r.HandleFunc("/graph", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
this one should probably be part of the `legacy` API package, right? Since it's supporting an old route?

Reply:
While I generally agree, the legacy package is actually mounted on `/api/v1/`. So adding it there won't work unless we mount that router to `/`. I'd say it's fine, there's basically nothing that handler is doing. :man_shrugging:

Reply:
I'm just worried about code sprawl, ie how long to we want to maintain this redirect. putting it here in the main may cement it as part of the API